test/test_expanded_weights.py

```diff
@@ -688,24 +688,22 @@ def filter_supported_tests(t):
     if 'constructor' not in test_param:
         name = test_param.pop('module_name')
         test_param['constructor'] = getattr(nn, name)
-    decorator = test_param.pop('decorator', None)
+    decorator = test_param.pop('decorator', lambda test: test)
     test = ContextManagerTests(**test_param)
     test_name = test.get_name()
     if hasattr(TestExpandedWeightModule, test_name):
         raise RuntimeError('Found two tests with the same name: ' + test_name)
     test_name_multi_input = test.get_name() + "_multiple_inputs"
     if hasattr(TestExpandedWeightModule, test_name_multi_input):
         raise RuntimeError('Found two tests with the same name: ' + test_name)
-    if decorator is not None:
-        fn = decorator(fn)  # noqa: F821
```

Comment:
I assume this was dead code since `fn` isn't even defined, lol

Should this be `test = decorator(test` instead?